doc/source/whatsnew/v1.2.0.rst

```diff
@@ -207,7 +207,7 @@ Performance improvements
 
 - Performance improvements when creating Series with dtype `str` or :class:`StringDtype` from array with many string elements (:issue:`36304`)
 - Performance improvement in :meth:`GroupBy.agg` with the ``numba`` engine (:issue:`35759`)
--
+- Performance improvements when creating `pd.Series.map` from a huge dictionary (:issue:`34717`)
```

Comment:
can you write this as ``` :meth:`pd.Series.map` ``` otherwise it wont' render

Reply:
Yes @jreback  i will sure add it.